kusto/ingest/internal/resources/resources.go

```diff
@@ -57,10 +57,6 @@ func parse(uri string) (*URI, error) {
 		return nil, fmt.Errorf("URI scheme must be 'https', was '%s'", u.Scheme)
 	}
 
-	if !strings.HasSuffix(u.Hostname(), ".core.windows.net") {
-		return nil, fmt.Errorf("URI hostname does not end with '.core.windows.net'")
-	}
-
 	hostSplit := strings.Split(u.Hostname(), ".")
 	if len(hostSplit) != 5 {
```

Comment:
the assumption here is _resource.resource_type.core.[dns suffix]_
The condition should better be changed to >4 cause we can't assure the DNS suffix for the cloud would only have two components.

Reply:
Fixed